packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuItemWrapper.types.ts

```diff
@@ -18,7 +19,7 @@ export interface IContextualMenuItemWrapperProps extends React.Props<IContextual
   /**
    * CSS class to apply to the context menu.
    */
-  classNames: IMenuItemClassNames;
```

Comment:
Unsure if this is breaking since `IProcessedStyleSet<IContextualMenuItemStyles>` is a superset of the fields in `IMenuItemClassNames`